server/core/src/irodsReServer.cpp

```diff
@@ -453,17 +482,16 @@ namespace {
         irods::at_scope_exit at_scope_exit{[&rule_exec_submit_inp] {
             freeBBuf(rule_exec_submit_inp.packedReiAndArgBBuf);
         }};
-
-        ix::client_connection conn;
-
+        
+        ix::client_connection conn = get_new_connection(std::nullopt);
         try {
             rule_exec_submit_inp = fill_rule_exec_submit_inp(conn, rule_id);
         }
         catch (const irods::exception& e) {
             irods::log(e);
             return;
         }
-
```

Comment:
lets restore this line since the code above it is not related to the code that follows it